phasespace/fulldecay/__init__.py

```diff
@@ -0,0 +1,15 @@
+import sys

```

Comment:
Not needed

Reply:
In this main submodule file I would personally have a good docstring explaining the scope. There's a lot to be taken from the discussions we had in that DecayLanguage issue https://github.com/scikit-hep/decaylanguage/issues/86.

Reply:
> Not needed

`sys` is used on line 13, similar to [hist](https://github.com/scikit-hep/hist/blob/8e2a85d70c922c6191ac8fba0bad1cbbd281cbb3/src/hist/plot.py#L14-L25).

> In this main submodule file I would personally have a good docstring explaining the scope. There's a lot to be taken from the discussions we had in that DecayLanguage issue [scikit-hep/decaylanguage#86](https://github.com/scikit-hep/decaylanguage/issues/86).

Good point, I will add an explanatory docstring.

Reply:
Ah, I managed to forget the stderr :D.